test/redirects.ts

```diff
@@ -459,10 +459,19 @@ test('method rewriting', withServer, async (t, server, got) => {
 		});
 		response.end();
 	});
+	server.get('/', (request, response) => {
+		request.pipe(response);
+	});
```

Comment:
Is this code really required?

Reply:
Also here, you're right. It's only needed on the post request.